app/models/admin_set.rb

```diff
@@ -0,0 +1,3 @@
+class AdminSet < ActiveFedora::Base
```

Comment:
Do you have a recommendation here?  Should I rename it to Sufia::AdminSet everywhere? Is there a more logical way to simply override CC's default `AdminSetBehavior` (which is the core problem here)?

If none of this sounds reasonable, I guess I could patch CC first (in its AdminSetBehavior) and then we'd have to use a new CC version in Sufia 7.3.0